python/lsst/ts/m2gui/control_tabs.py

```diff
@@ -0,0 +1,79 @@
+# This file is part of ts_m2gui.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["ControlTabs"]
+
+from PySide2.QtWidgets import QListWidget
+from PySide2.QtCore import Qt
+
+
+class ControlTabs(QListWidget):
+    """Control tables.
+
+    Parameters
+    ----------
+    log : `logging.Logger`
+        A logger.
+
+    Attributes
+    ----------
+    log : `logging.Logger`
+        A logger.
+    """
+
+    def __init__(self, log):
+        super().__init__()
+
+        self.log = log
```

Comment:
self.log isn't used and I fear there isn't need to use it. I recommend to change strategy for next phase. If you need logging, use Python logging module, so you will not need to pass reference for log.

Reply:
Will use it in the next ticket. It is python `logging` module. Thanks!